hardhat.config.ts

```diff
@@ -1,7 +1,12 @@
-/** @type import('hardhat/config').HardhatUserConfig */
-require('dotenv').config();
+import dotenv from 'dotenv';
+import { HardhatUserConfig } from "hardhat/config";
+import '@typechain/hardhat'
+import '@nomicfoundation/hardhat-ethers'
+import "@nomicfoundation/hardhat-verify";
 
-module.exports = {
+dotenv.config();
+
+const config: HardhatUserConfig = {
```

Comment:
Minor issue but I think hardhat uses a chainId of 31337

Reply:
Also do we want to prepopulate the other chains Ajna is deployed to?

Reply:
Corrected this in commit 9950397f0390a4879cb410e16af923f838001c0f.  We currently only use hardhat to run automated tests; the keeper itself works with just ethers.js.  I only meddled with the file in this PR because I made an (unsucessful) attempt to get conract verification working.